packages/teams-js/src/private/files.ts

```diff
@@ -297,6 +297,28 @@ export namespace files {
     extension: string;
   }
 
+  /**
+   * Object used to represent a file
+   */
+  export interface File extends Blob {
+    /**
+     * Last modified timestamp
+     */
+    lastModified: Date;
+    /**
+     * Name of the file
+     */
+    name: string;
+    /**
+     * The file path.
+     */
+    webkitRelativePath: string;
+    /**
+     * Represents if file or a folder
+     */
+    isFolder?: boolean;
```

Comment:
I appreciate you making a well-documented interface for this, but we shouldn't have a function called "isFolder" on a file. Files are not folders.

Reply:
Removed the field **isFolder**, as it was added to make the interface common for file and folder. Removed it as per your comments. If needed in future, will look for generic alternatives.